provider/cmd/main.go

```diff
@@ -13,9 +12,10 @@ import (
 
 func main() {
 	ctx := ctrl.SetupSignalHandler()
+	log := ctrl.Log.WithName("main")
 
 	if err := app.Command().ExecuteContext(ctx); err != nil {
-		fmt.Println(err.Error())
+		log.V(1).Error(err, "Error running libvirt provider")
```

Comment:
```suggestion
		log.V(1).Error(err, "error running libvirt provider")
```

i prefer same format of error messages.

Reply:
fixed